helpers/liquity.ts

```diff
@@ -63,3 +62,103 @@ export function liquityV2Exports(config: IJSON<LiquityV2Config>) {
   return { adapter: exportObject, version: 2 } as SimpleAdapter
 }
 
+const RedemptionEvent = 'event Redemption(uint _attemptedLUSDAmount, uint _actualLUSDAmount, uint _ETHSent, uint _ETHFee)'
+const BorrowingEvent = 'event LUSDBorrowingFeePaid(address indexed _borrower, uint _LUSDFee)'
+const liquidationEvent = 'event Liquidation(uint _liquidatedDebt, uint _liquidatedColl, uint _collGasCompensation, uint _LUSDGasCompensation)'
+
+type LiquityV1Config = {
+  troveManager: string
+  stableCoin: string
+  holderRevenuePercentage?: number
+  protocolRevenuePercentage?: number
+  redemptionEvent?: string
+  borrowingEvent?: string
+}
+
+
+
+export const getLiquityV1LogAdapter: any = (config: LiquityV1Config): FetchV2 => {
+  const fetch: FetchV2 = async (fetchOptions) => {
+    const { createBalances, getLogs, api } = fetchOptions
+    const dailyFees = createBalances()
+
+    const redemptionEvent = config.redemptionEvent || RedemptionEvent
+    const borrowingEvent = config.borrowingEvent || BorrowingEvent
+
+    // Get brrower opertaor contract
+    const borrowerOperator = await api.call({ abi: 'address:borrowerOperationsAddress', target: config.troveManager })
+    
+    // redemtions fees
+    const redemptionLogs = await getLogs({
+      target: config.troveManager,
+      eventAbi: redemptionEvent,
+    })
+
+    // liquidations logs 
+    const liquidationLogs = await getLogs({
+      target: config.troveManager,
+      eventAbi: liquidationEvent,
+    })
+
+    // event LUSDBorrowingFeePaid(address indexed _borrower, uint _LUSDFee);
+    const borrowingLogs = await getLogs({
+      target: borrowerOperator,
+      eventAbi: borrowingEvent,
+    })
+
+    // get _ETHFee from event
+    redemptionLogs.forEach((logs) => {
+      dailyFees.addGasToken(BigInt(logs['_ETHFee']))
+    })
+
+    // get _LUSDFee from event
+    borrowingLogs.forEach((logs) => {
+      dailyFees.add(config.stableCoin, BigInt(logs['_LUSDFee']))
+    })
+
+    // get _LUSDGasCompensation from event
+    liquidationLogs.forEach((logs) => {
```

Comment:
can move this liquidation log data to the bottom, as they should not be included as part of daily revenue, it goes to the stability pool right which is daily supply sideRevenue? (I might be wrong)

Reply:
yes value ETH go to The Stability Pool when got liquidate, i guess it should count only supplySide